lib/config.js

```diff
@@ -9,7 +9,8 @@ const coffee = require('coffee-script');
 const debug = require('debug')('brunch:config');
 const deepAssign = require('deep-assign');
 
-const loadInit = require('deppack').loadInit;
+const deppack = require('deppack'); // isNpm
```

Comment:
@paulmillr the `isNpm` call **has** to happen on the deppack object, don't change it to `const isNpm = require('deppack').isNpm;`